### PR TITLE
Rename `ceph-salt deploy` to `ceph-salt apply`

### LINKED ISSUE #154: Rename `ceph-salt deploy` to `ceph-salt apply`
What `ceph-salt deploy [minion_id]` do is:
1) provision host(s)
2) if running for the first time
2.1) bootstrap a new minimal ceph cluster (1 Mon and 1 Mgr), 
2.2) configure Ceph orchestrator

The rest of the ceph cluster deployment must be done by `ceph orch ...`, so I believe `ceph-salt apply` is a better name than `ceph-salt deploy`.

### PR DESCRIPTION
**Pull request by ricardoasmarques**
Fixes: https://github.com/ceph/ceph-salt/issues/154

Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

File: ceph_salt/apply.py [hunk 1]
```diff
@@ -1214,13 +1214,13 @@ def check_formula():
                 PP.pl_red('Failed to restart salt-master service, please restart it manually')
                 return 3
 
-        # checking ceph-salt again after salt-master restart
+        # checking ceph-salt formula again after salt-master restart
```
Comment (smithfarm) [inline ceph_salt/apply.py]: s/checking/check/

File: ceph_salt/apply.py [hunk 2]
```diff
@@ -1235,34 +1235,35 @@ def check_formula():
...
         deployed = len(host_ls) > 0
 
         # day 1, but minion_id specified
         if minion_id is not None and not deployed:
-            logger.error("cluster not deployed and minion_id provided")
-            PP.pl_red("Cluster is not deployed yet, please apply the deployment to "
-                      "all minions at the same time: \"ceph-salt deploy\"")
+            logger.error("ceph cluster not deployed and minion_id provided")
+            PP.pl_red("Ceph cluster is not deployed yet, please apply the config to "
+                      "all minions at the same time to boostrap a new Ceph cluster: "
```
Comment (smithfarm) [inline ceph_salt/apply.py]: s/boostrap/bootstrap/

File: ceph_salt/apply.py [hunk 3]
```diff
@@ -1235,34 +1235,35 @@ def check_formula():
...
-                    logger.error("minion_id already deployed: %s", minion_id)
-                    PP.pl_red("Minion '{}' is already deployed".format(minion_id))
+                    logger.error("minion_id already managed by cephadm: %s", minion_id)
+                    PP.pl_red("Minion '{}' is already managed by cephadm".format(minion_id))
                     return 8
         return 0
 
     @staticmethod
-    def check_deploy_prerequesites(minion_id):
+    def check_apply_prerequesites(minion_id):
```
Comment (smithfarm) [inline ceph_salt/apply.py]: s/prerequesites/prerequisites/

File: ceph_salt/apply.py [hunk 4]
```diff
@@ -1285,17 +1286,17 @@ def check_deploy_prerequesites(minion_id):
...
             return retcode
 
-        PP.println("Ready for deployment!")
+        PP.println("Ready to apply config!")
         return 0
 
     def run(self):
         # validate
-        retcode = self.check_deploy_prerequesites(self.minion_id)
+        retcode = self.check_apply_prerequesites(self.minion_id)
```
Comment (smithfarm) [inline ceph_salt/apply.py]: s/prerequesites/prerequisites/

**Review by smithfarm — APPROVED** [review]
modulo some grammar/spelling nits

**Comment by ricardoasmarques** [comment]
@smithfarm fixed typos, thanks.

**Comment by smithfarm** [comment]
@ricardoasmarques We'll need to coordinate ceph-salt and sesdev releases for this change.

**Pull request merged**